### PR TITLE
Handle WorkerMessageFailedEvent

### LINKED ISSUE #88: Listen to WorkerMessageFailedEvent
### Expected behavior
Exceptions thrown in `MessageHandlerInterface` are catched by the `BugsnagListener`.

### Observed behavior
This isn't the case.

### Steps to reproduce
Have a `MessageHandlerInterface` throw an exception.

### Version
- bugsnag/bugsnag v3.18.0
- bugsnag/bugsnag-symfony v1.5.1

I suspect all versions are impacted though.

### Additional information
Every exception thrown inside a message handler is catched by the worker so it can continue working so there is no `kernel.exception`. Instead a `WorkerMessageFailedEvent` is dispatched so the `BugsnagListener` should also listen to it.

### PR DESCRIPTION
**Pull request by MatTheCat**
## Goal

Fix #88 

I don't know yet how to test this but I wanna be sure I've got the right approach.

### CODE REVIEW

**Comment by rjharrison** [comment]
Thanks for the code Mat. Looks good. I think we'd use notifyException() rather than notify(). We'll have a discussion internally about whether this is a feature that we want to add, and get back to you.

**Comment by MatTheCat** [comment]
Why would you wilfully ignore exceptions?

**Comment by rjharrison** [comment]
> Why would you wilfully ignore exceptions?

We aim to report as much as possible. I think the question is more about how we do this, rather than _if_ we want to do this. We will be investigating this further.

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
         "symfony/console": "^2.7|^3.0|^4.0",
         "symfony/http-foundation": "^2.7|^3.0|^4.0",
         "symfony/security": "^2.7|^3.0|^4.0",
-        "bugsnag/bugsnag": "^3.15.0"
+        "bugsnag/bugsnag": "^3.15.0",
+        "symfony/messenger": "^4.3"
```
Comment (damienalexandre) [inline composer.json]: This component cannot be installed with Symfony < 4.3:

composer conflicts:
- symfony/event-dispatcher: <4.3
- symfony/debug: <4.1

Comment (MatTheCat) [reply composer.json]: Shouldn't it be a dev dependency?

**Pull request closed without merging**